src/js/core/directives/ui-grid-menu.js

```diff
@@ -99,17 +97,22 @@ function ($compile, $timeout, $window, $document, gridUtil, uiGridConstants, i18
 
         // Turn off an existing document click handler
         angular.element(document).off('click touchstart', applyHideMenu);
+        $elm.off('keyup', checkKeyUp);
+        $elm.off('keydown', checkKeyDown);
 
         // Turn on the document click handler, but in a timeout so it doesn't apply to THIS click if there is one
         $timeout(function() {
           angular.element(document).on(docEventType, applyHideMenu);
+          $elm.on('keyup', checkKeyUp);
+          $elm.on('keydown', checkKeyDown);
+
         });
         //automatically set the focus to the first button element in the now open menu.
         gridUtil.focus.bySelector($elm, 'button[type=button]', true);
       };
 
 
-      $scope.hideMenu = function(event, args) {
+      $scope.hideMenu = function(event) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ["event" is defined but never used](https://www.codacy.com/app/Angular-UI-Grid/ui-grid/file/1998102727/issues/source?bid=2955261&fileBranchId=3118992#l115)